c_glib/README.md

```diff
@@ -251,3 +251,28 @@ gobject-introspection requires libffi, and it's automatically installed with gob
 ### build failed - /usr/bin/ld: cannot find -larrow
 
 Arrow C++ must be installed to build Arrow GLib. Run `make install` on Arrow C++ build directory. In addtion, on linux, you may need to run `sudo ldconfig`.
+
+### build failed - unable to load http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl
+
+On OSX you may need to set the following environment variable:
```

Comment:
Can you use "macOS" instead of "OSX"? We use the latest OS name in this document.